src/sections/Header/index.module.scss

```diff
@@ -27,7 +27,7 @@
 .header {
   background: var(--color-fill-app);
   border-bottom: var(--space-01) solid var(--black2);
-  position: sticky;
+  position: fix;
```

Comment:
Please revert that. There's no such "position: fix".

Reply:
Also, the navbar is not broken.

Reply:
oh, it's supposed to be `fixed` but for some reason it ended up being `fix` and I have no idea how that happened. Changing features to only display one per line only masked the underlying issue of it not reaching the other end, especially on smaller viewports while it being `fixed` always ensures that. Do you want me to change it back to sticky or update it to fixed? Extremely sorry about "fix", though

Reply:
As I mentioned, it should be sticky 👀

Reply:
I reverted it back to sticky